supervision/dataset/core.py

```diff
@@ -96,6 +97,28 @@ def from_pascal_voc(
 
         Returns:
             Dataset: A Dataset instance containing the loaded images and annotations.
+
+        Example:
+            ```python
+            >>> import roboflow
+            >>> from roboflow import Roboflow
+            >>> import supervision as sv
+
+            >>> roboflow.login()
+
+            >>> rf = Roboflow()
+
+            >>> project = rf.workspace(WORKSPACE_ID).project(PROJECT_ID)
+            >>> dataset = project.version(PROJECT_VERSION).download("voc")
+
+            >>> train_dataset = sv.Dataset.from_yolo(
```

Comment:
🔥 splits are separate sv Datasets?

Reply:
Yeah. You load every split as a separate `sv.Dataset` but that is pretty much the standard approach. Torch Datasets and Dataloaders work the same way. But I think we will build some utils to split and merge datasets to.